app/generic/verifier/facades/acapy_verifier_v1.py

```diff
@@ -45,6 +45,51 @@ async def create_proof_request(
         )
         return utils.record_to_model(presentation_exchange)
 
+    @classmethod
+    async def get_creds(cls, controller: AcaPyClient, pres_ex_id: str):
```

Comment:
In `acapy_verifier` it is called `get_credentials` and here `get_creds`? 

Maybe call it `get_credentials_for_request` to be as explicit as we can be